CHANGELOG

```diff
@@ -1,3 +1,7 @@
+v1.1.0:
```

Comment:
Please leave all version changes to the person who will make the release if this is merged.